articles/openshift/howto-deploy-java-eap-app.md

```diff
@@ -0,0 +1,395 @@
+---
+title: Deploy a Java application with Red Hat JBoss Enterprise Application Platform (JBoss EAP) on an Azure Red Hat OpenShift 4 cluster
+description: Deploy a Java application with Red Hat JBoss Enterprise Application Platform on an Azure Red Hat OpenShift 4 cluster.
+author: yborgess
+ms.date: {@date}
+ms.topic: article
+keywords: java, jakartaee, microprofile, EAP, JBoss EAP, ARO, OpenShift, JBoss Enterprise Application Platform
+---
+
+# Deploy a Java application with Red Hat JBoss Enterprise Application Platform on an Azure Red Hat OpenShift 4 cluster
+
+This guide demonstrates how to deploy a Jakarta EE 8 application running on Red Hat JBoss Enterprise Application Platform (JBoss EAP) that connects to a Microsoft SQL Server to an Azure Red Hat OpenShift (ARO) 4 cluster by using [EAP Helm Charts](https://jbossas.github.io/eap-charts).
```

Comment:
I struggled with this sentence yesterday but didn't have any better ideas that didn't amount kind of duplicating the next paragraph. And I think a single sentence summary is a good thing.

Maybe this will help...

Put a comma after "Jakarta EE 8 application" and after "Microsoft SQL Server".  That separates off the entire "running on Red Hat JBoss Enterprise Application Platform (JBoss EAP) that connects to a Microsoft SQL Server" part, which helps. That block can be removed entirely and you have a grammatically correct and accurate sentence. But of course you don't want to remove it! So separating it a bit with commas might help.

Reply:
Done